pytest_django/live_server_helper.py

```diff
@@ -24,7 +24,10 @@ def __init__(self, addr: str) -> None:
                 and conn.settings_dict["NAME"] == ":memory:"
             ):
                 # Explicitly enable thread-shareability for this connection
-                conn.allow_thread_sharing = True
+                try:
+                    conn.inc_thread_sharing()
+                except AttributeError:  # Django < 3
```

Comment:
Instead of catching an `AttributeError`, let's do a django version check instead. This is easier find and remove once we drop support for these Django versions.